From c1843df7ab37fda4f9bb9003f4b5f7b939b41813 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 6 May 2019 21:21:58 -0600
Subject: [PATCH 4/4] [SQUASH][DNM] Merge tag 'android-9.0.0_r37' into
 staging/lineage-16.0_merge-android-9.0.0_r37

Android 9.0.0 Release 37 (PQ3A.190505.002)

* tag 'android-9.0.0_r37':
  Version bump to PQ3A.190505.002
  [DO NOT MERGE] Update Platform Security string to 2019-05-05 for Pi-dev Bug:128322951 (cherry picked from commit 7d44fc8d59547bc583bad61fe2cae9fa14d31baf)
  [DO NOT MERGE] Update Security String to 2019-05-01 in pi-dev Bug: 128322951 (cherry picked from commit 1159bb8b17df6caf1ce075d120c811b17eb73554)
  Version bump to PQ3A.190220.002
  [DO NOT MERGE] Update platform security string to 2019-04-01 on oc-mr1-dev Bug:124119313
  Update Security String to 2019-04-05 Bug: 124119313
  Update Security String to 2019-04-01 Bug: 124119313
  Version bump to PQ3A.190122.001
  [DO NOT MERGE] Update Platform Security String to 2019-03-05 for oc-mr1-dev Bug:122541777
  [DO NOT MERGE] Update Platform Security String to 2019-03-01 for oc-mr1-dev Bug:122541777
  [DO NOT MERGE] Update Platform Security String to 2019-03-01 for nyc-dev Bug:122541777
  Update Platform Security String to 2019-03-05 for pi-dev Bug:122541777
  Update Platform Security String to 2019-03-01 for pi-dev Bug:122541777
  [DO NOT MERGE] Security String update to 2019-02-05 Bug: 120144419
  [DO NOT MERGE] Security String update to 2019-02-01 Bug: 120144419
  [DO NOT MERGE] Security String update to 2019-02-05 Bug: 120144419
  [DO NOT MERGE] Security String update to 2019-02-01 Bug: 120144419

https://github.com/haggertk/android_build/tree/lineage-16.0_merge-android-9.0.0_r37

Change-Id: Id9ec8ccc87eeb9c4e434c18ffbe4e52c8c98afa0
---
 core/build_id.mk         | 2 +-
 core/version_defaults.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index 94db1368e..4c46c5cf1 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=PQ2A.190405.003
+export BUILD_ID=PQ3A.190505.002
diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 1df6fc5d5..80e5007ea 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -231,7 +231,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2019-04-05
+      PLATFORM_SECURITY_PATCH := 2019-05-05
 endif
 
 ifndef PLATFORM_SECURITY_PATCH_TIMESTAMP
-- 
2.17.1

